amundsen_application/static/js/components/common/Table/index.tsx

```diff
@@ -37,13 +41,15 @@ export interface TableOptions {
 }
 
 export interface TableProps {
+  data: RowData[];
   columns: TableColumn[];
-  data: any[];
   options?: TableOptions;
 }
 
 const DEFAULT_EMPTY_MESSAGE = 'No Results';
 const EXPAND_ROW_TEXT = 'Expand Row';
+const INVALID_DATA_ERROR_MESSAGE =
```

Comment:
is the `Error` word needed here?

if the Error is serialized ti will add the `Error` string by itself.

Reply:
Makes sense, thanks!